proto/xudrpc.proto

```diff
@@ -467,6 +467,15 @@ message Order {
   uint64 hold = 10 [json_name = "hold"];
 }
 
+message Currency {
+  // Ticker symbol of this currency (e.g. BTC)
+  string ticker_symbol = 1 [json_name = "ticker_symbol"];
+  // The places to the right of the decimal point of the smallest subunit (e.g. 8, 18)
+  uint64 digits = 2 [json_name = "digits"];
+  // Global identifier of this currency
+  string global_identifier = 3 [json_name = "global_identifier"];
```

Comment:
For LND currencies, we don't actually use a "global" identifier, just descriptors like "bitcoin-mainnet". So maybe let's just call this `identifier` and drop the global references.